2019.08.09_expectations_for_an_event_store.md

```diff
@@ -19,6 +19,8 @@ What I expect from some system managing storage of streams of events, intended t
 - ability to have metadata on stream level even if the stream does not exists yet
 - ability to have metadata on events
 - idempotent writes (more on that later)
+- support for high-availability scenarios
+- support for at least write quorums and causally consistent, monotonic reads
```

Comment:
Are those must have or really , _really_ nive to have

Reply:
It depends on perspective tbh. In production environment, such a setup can provide availability which is more that just nice to have; I am not aware of any server product that runs on a single instance of a database.

Reply:
What I'm trying to say with the above is that this is a must-have guarantee from a database for a lot of services needing to store information. At the same time, it's not needed at all (it's not even a nice to have) for other (rarer ime) scenarios.

I'm happy to move that to bonuses if you prefer.